packages/qtile-git/qtile-git.pacscript

```diff
@@ -0,0 +1,44 @@
+name="qtile-git"
+pkgname="qtile"
+url="https://github.com/qtile/qtile.git"
+license="MIT"
+depends="python3-cffi xserver-xorg python3-xcffib python3-cairocffi libpangocairo-1.0-0 python3-pip python3-dbus libpulse-dev pavucontrol"
+description="A full-featured, hackable tiling window manager written and configured in Python"
+maintainer="Ruturajn <nanotiruturaj@gmail.com>"
+
+pkgver() {
+  git ls-remote "${url}" master | cut -f1
+}
+version="$(pkgver)"
+
+prepare() {
+  true
+}
+
+build() {
+  true
+}
+
+install() {
+  pip install --upgrade --no-cache --force-reinstall cairocffi
+  pip3 install dbus-next
+  pip3 install .
+  sudo python3 setup.py install
+  sudo mkdir -p /usr/src/pacstall/qtile-git
+  sudo cp -r ../qtile /usr/src/pacstall/qtile-git
+  sudo touch /usr/share/xsessions/qtile.desktop
```

Comment:
This should be `/usr/src/pacstall/qtile-git/usr/share/xessions/qtile.desktop`

Reply:
Ooh. I remember I had to do similar thing in [awesome-git](https://github.com/pacstall/pacstall-programs/blob/master/packages/awesome-git/awesome-git.pacscript) too

Reply:
Never mind, it's showing up now. I will test it again though and then commit.